external/Makefile

```diff
@@ -97,6 +100,7 @@ include netlib/Make.defs
 include json/Make.defs
 include mbedtls/Make.defs
 
+
```

Comment:
Please, remove this additional empty line.

Reply:
I removed the addtional line